### DISCUSSION TITLE
Need help with a project

### DISCUSSION THREAD

**Discussion by chatiktt** [community/community#145112] [Programming Help]
Need help with a project.

I as a person who is inexperienced and has no worthwhile programming skills I am looking for help.
I study in college and we need to do a project, I was offered to take a ready-made game from the platform itch.io and remake under myself, only I do not know how to get the internal code of those or other games. 
I read about it, they say you need to decompile, but how and through what to do I do not understand.
Have been looking for information on this, haven't found anything.

Those who understand this help!

**Comment by denis23x** [comment]
Hi @chatiktt 

As you mentioned, for person with no skills it will not help you to make a project, even if you get source code. Also, it is often impossible and/or illegal to use.

I would advise you to make your project from scratch (with training along the way), here is a great and easy to use game engine for this purpose https://docs.godotengine.org/en/stable/getting_started/introduction/introduction_to_godot.html

There are many tutorials on YouTube

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @chatiktt!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
